### PR TITLE
Empty arraycollections should be serialized as empty array

### PR DESCRIPTION
**Pull request by Keeo**

### CODE REVIEW

File: Services/EmberDataSerializerManager.php [hunk 1]
```diff
@@ -208,9 +208,9 @@ private function parseSerializableObject(EmberDataSerializableInterface $object,
 
                 } else {
                     if ($plural) {
-                        $this->data[$adapter->getModelNamePlural()][$index][$key] = $value;
+                        $this->data[$adapter->getModelNamePlural()][$index][$key] = count($value) === 0 ? [] : $value;
```
Comment (UniqueLibs) [inline Services/EmberDataSerializerManager.php]: Please do not use short array syntax to stay compatible with php 5.3

**Comment by UniqueLibs** [comment]
Please change the short array syntax to stay compatible with php 5.3 and then i will merge your pull request.
Thanks for your help.

**Comment by Keeo** [comment]
Done. Although I am not happy with this approach because there is similar issue with \DateTime being serialized as object and not iso. It feels like there should be some kind of general transformer, not these hotfixes.

**Comment by UniqueLibs** [comment]
You should use the http://jmsyst.com/bundles/JMSSerializerBundle to handle the result.
I think this would solve most of your output problems, like the problem with \DateTime objects.

In addition you could use the FOSRestBundle.
I use this bundle too, thats why I did not had that kind of problems.

**Pull request merged**